core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowLocalRunnerPlugin.scala

```diff
@@ -419,7 +420,14 @@ object CloudflowLocalRunnerPlugin extends AutoPlugin {
     updatedStreamlets.map(streamlets => applicationDescriptor.copy(streamlets = streamlets))
   }
 
-  def runPipelineJVM(applicationDescriptorFile: Path,
+  //side-effects
+  def nextDebugPort() = {
```

Comment:
I'm not sure this is a good idea, can we bind it to an sbt SettingKey?

Reply:
A SettingKey for each port? How would you write the user story?

Reply:
Got it now ... 
But in this case, we are never resetting the value in a single sbt session that might become problematic ...
I would say, one settingKey for the `baseDebugPort` and make sure it resets from one `runLocal` to the next.

Reply:
very nice catch 👍

Reply:
done